Lib/test/test_symtable.py

```diff
@@ -319,7 +319,8 @@ def test_symbol_repr(self):
                                 "       def method():\n"
                                 "           return x\n",
                                 "?", "exec")
-        func_f = st3.get_children()[0]
+        # child 0 is for __annotate__
+        func_f = st3.get_children()[1]
```

Comment:
Maybe use `lookup()` instead of `get_children()`?

Reply:
Using `lookup` gives me an instance of `Symbol` but I want the symbol table of 'f' and then get the symbol table of 'A' and finally lookup for the symbol 'x'. 

It's something that I wanted to ask as well, but would it make sense to have a better find interface where we could also find children instead of getting them as a list?

Reply:
Oh, I see. I think that the current interface is fine, taking into account that there may be several symbol tables with the same name, so we would need to add methods for combinations `lookup()`+`get_namespace()` and `lookup()`+`get_namespaces()`.

Reply:
Yes, that's why I was a bit conflicted about whether implementing a better find function or not. It's a bit annoying to use `get_children()[i]` but it's only for the tests so I can live with that.